src/pal/src/exception/seh-unwind.cpp

```diff
@@ -195,6 +198,9 @@ BOOL PAL_VirtualUnwind(CONTEXT *context, KNONVOLATILE_CONTEXT_POINTERS *contextP
 
     return TRUE;
 }
+#else
+#error don't know how to unwind on this platform
```

Comment:
Do you think that it might be useful when compiling on Linux to print a message with a suggestion how to fix it? Something like "please make sure the libunwind8 and libunwind8-dev packages are installed".